### PR TITLE
 rasm2 wrong code instead of error

### PR DESCRIPTION
**Pull request by McLovi9**

### CODE REVIEW

File: t.asm/arm/arm [hunk 1]
```diff
@@ -197,4 +197,5 @@ BROKEN=1
 ARGS="-a arm -b 16"
 CMDS="pa b lr"
 EXPECT=''
+EXPECT_ERR='Invalid-instruction'
```
Comment (radare) [inline t.asm/arm/arm]: looks wrong to me

Comment (McLovi9) [inline t.asm/arm/arm]: whats wrong ?

Comment (radare) [inline t.asm/arm/arm]: Invalid-instruction is not what its expected. Well as far as its marked as broken it can be ok i guess

Comment (McLovi9) [inline t.asm/arm/arm]: OK, I can remove it if you want.

**Pull request merged**